src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawerAdapter.java

```diff
@@ -8,11 +8,13 @@
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class DrawerAdapter extends DrawPanelController implements Job2dDriver {
```

Comment:
The name is better but it lacks a driver aspect.

Inheritance is not used (no super drawLine calls). Should be removed!